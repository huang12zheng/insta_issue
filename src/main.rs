use insta::assert_debug_snapshot;

fn main() {
    println!("Hello, world!");
}

#[test]
fn test_snapshots() {
    // let value = vec![1, 2, 3];
    // assert_debug_snapshot!(value);
    insta::assert_snapshot!("ğŸ«—", "ğŸ«—");
}
