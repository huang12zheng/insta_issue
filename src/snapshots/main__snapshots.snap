---
source: src/main.rs
expression: "\"ğŸ«—\""
---
ğŸ«—
